processor.py

```diff
@@ -265,7 +251,7 @@ def go_single(self, url: str) -> None:
                             ptype: str = getattr(p, "PROCESSOR_TYPE")
                             assert ptype in _PROCESSOR_TYPES, "Unknown processor type"
                             if ptype == _PROCESSOR_TYPE_TREE:
-                                tree.process(session, p)
+                                tree.process(session, vars(p))
```

Comment:
Is it still logical to have the `token_container.process()` call below using the old scheme (p instead of vars(p))?

Reply:
Not really, I think it would make most sense to have the module types all be mappings or `ProcEnv`s. I'll add some fixes.